### PR TITLE
fix: github discussions references

### PR DESCRIPTION
**Pull request by danieldietzler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@discordx/importer": "^1.2.3",
+    "@octokit/request-error": "^5.0.1",
```
Comment (bo0tzz) [inline package.json:21]: Why is this a separate package lol?

**Review by bo0tzz — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (danieldietzler) [reply package.json:21]: Yeah idk... There is no error type in the main package. It's stupid imo

Comment (danieldietzler) [reply package.json:21]: I didn't want to go with any though if they already provide types

File: src/events/messageEvents.ts [hunk 2]
```diff
@@ -60,14 +61,14 @@ export class MessageEvents {
             issue_number: Number(id),
           });
 
-          if (response.status === 200) {
-            const type = response.data.pull_request ? 'PR' : 'ISSUE';
-            links.add(`[${type}] ${response.data.title} ([#${id}](${response.data.html_url}))`);
+          const type = response.data.pull_request ? 'PR' : 'ISSUE';
```
Comment (jrasm91) [inline src/events/messageEvents.ts:64]: all the try/catch stuff looks a bit messy to me. Maybe you could:
- If no matches, continue (and de-indent everything - top of the for loop)
- Move the github requests to private functions that return a string or null.
- Update the for loop to just be `const link = await this.getIssueOrPR(id) || await this.getDiscussion(id);`

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**